deploy/device-operator.yaml

```diff
@@ -111,6 +111,17 @@ spec:
           description: VolStatus string that specifies the current state of the volume
             provisioning request.
           properties:
+            error:
```

Comment:
I am not sure if this field  complies with API [conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties) . Maybe we can have a `Ready` field similar to pods. Possible values being `true`, `false` and `unknown`. To indicate an error code, we can utilize the `Reason` field and `Message` field would have the verbose error